keypoint_detection/train/train.py

```diff
@@ -88,7 +103,27 @@ def parse_channel_configuration(channel_configuration: str) -> List[List[str]]:
     parser = add_system_args(parser)
     parser = KeypointDetector.add_model_argparse_args(parser)
     parser = Trainer.add_argparse_args(parser)
-    parser = COCOKeypointsDataset.add_argparse_args(parser)
+
+    datasets_group = parser.add_argument_group("Datasets")
+    datasets_group.add_argument(
+        "--json_dataset_path",
+        type=str,
+        help="Absolute path to the json file that defines the train dataset according to the COCO format.",
+        required=True,
+    )
+    datasets_group.add_argument(
+        "--json_validation_dataset_path",
+        type=str,
+        help="Absolute path to the json file that defines the validation dataset according to the COCO format. \
+            If not specified, the train dataset will be split to create a validation set.",
+    )
+    datasets_group.add_argument(
+        "--json_test_dataset_path",
+        type=str,
+        help="Absolute path to the json file that defines the test dataset according to the COCO format. \
+            If not specified, no test set evaluation will be performed at the end of training.",
+    )
+
     parser = KeypointsDataModule.add_argparse_args(parser)
```

Comment:
Maybe have datamodule take the 3 strings and instantiate them? then the argparse arguments can be moved to the DataModule as well as the logic for dealing with the optional datasets